### PR TITLE
Feat/2.5.1

### PR DESCRIPTION
**Pull request by Noxemia**

### CODE REVIEW

**Comment by Noxemia** [comment]
This includes 2.5.2 cuz me and git be big friendos

File: packages/frontend/.env.example [hunk 1]
```diff
@@ -1,9 +1,9 @@
 GRAPHQL_API_ENDPOINT=https://localhost:8080/graphql
 
-FACEBOOK_OAUTH2_CLIENT_ID=
+FACEBOOK_OAUTH2_CLIENT_ID=2696370720593869
```
Comment (rosengrenen) [inline packages/frontend/.env.example]: why u leak my oauth credentials :c

**Review by rosengrenen — CHANGES_REQUESTED** [review]
remove oauth creds from env example

**Comment by Noxemia** [comment]
men kolla vad snyggt de e

**Review by rosengrenen — APPROVED** [review]

**Review by SantaPoro — APPROVED** [review]

**Pull request merged**